src/cell.rs

```diff
@@ -0,0 +1,24 @@
+use rand::{thread_rng, Rng};
+
+#[derive(Copy, Clone, PartialEq, Debug)]
+pub struct Pos {
+    pub x: i32,
+    pub y: i32,
+}
+
+#[derive(Clone, PartialEq, Debug)]
+pub struct Cell {
+    pub pos: Pos,
+    pub is_dead: bool,
+}
```

Comment:
Position can still be inferred from the indices in `CellMatrix`.

Reply:
You are right, I'll push it now